smurfs/src/actions/index.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+export const FETCHING_SMURFS_START = "FETCHING_SMURFS_START";
+export const FETCHING_SMURFS_SUCCESS = "FETCHING_SMURFS_SUCCESS";
+
+
+export const getSmurfs = () => dispatch => {
+    dispatch({ type: FETCHING_SMURFS_START });
+    axios
+      .get("http://localhost:3333/smurfs")
+      .then(res => {
+        dispatch({ type: FETCHING_SMURFS_SUCCESS, payload: res.data });
+      })
+      // .catch(err => {
```

Comment:
I wish you would have tried axios.post here
(
export const getSmurfs = smurf => dispatch => {
    dispatch({ type: ADD_SMURFS_START });
    axios
      .post("http://localhost:3333/smurfs", smurf)
      .then(res => {
        dispatch({ type: ADD_SMURFS_SUCCESS, payload: res.data });
      })
)